scrapinghub/client.py

```diff
@@ -284,15 +285,15 @@ class Spider(object):
 
         >>> project.spiders.get('spider2')
         <scrapinghub.client.Spider at 0x106ee3748>
-        >>> spider.id
+        >>> spider.key
         2
```

Comment:
I've updated all the docstrings with the following model:
- projects 123 and 456
- project 123 has 2 spiders: (1, spider1) and (2, spider2)